### PR TITLE
Validation support: python pyopengl enable for test case creation on meta-iotqa

### PR DESCRIPTION
**Pull request by YinThong**
Add python opengl dependency on computer vision profile test.

### CODE REVIEW

**Comment by YinThong** [comment]
test this please

**Comment by okartau** [comment]
ok to test

File: meta-refkit-computervision/recipes-devtools/opengl/python-pyopengl_3.0.1.bb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+LICENSE = "BSD"
+SRCNAME = "PyOpenGL"
+LIC_FILES_CHKSUM = "file://license.txt;md5=35e5f53336fec095a8a3f229568e151b"
+
+SRC_URI = "\
+  http://pypi.python.org/packages/source/P/${SRCNAME}/${SRCNAME}-${PV}.tar.gz \
+"
+S = "${WORKDIR}/${SRCNAME}-${PV}"
+
+inherit distutils
```
Comment (ipuustin) [inline meta-refkit-computervision/recipes-devtools/opengl/python-pyopengl_3.0.1.bb]: You ought to use Python 3. We aim to keep the various components scriptable with a single scripting language (Python 3) to make the interoperability easier.

File: meta-refkit/conf/distro/include/refkit-supported-recipes.txt [hunk 2]
```diff
@@ -29,9 +29,11 @@
 # command like this:
 # $ LC_ALL=C ex +'32,$ !sort -g' -scxa refkit-supported-recipes.txt
 
+
```
Comment (ipuustin) [inline meta-refkit/conf/distro/include/refkit-supported-recipes.txt]: Extra line.

**Comment by ipuustin** [comment]
This PR does much more than what's mentioned in the PR. Is the `[TEST_BUILD]` commit supposed to be here?

File: meta-refkit-computervision/recipes-devtools/opengl/python-pyopengl_3.0.1.bb (hunk 1 above)
Comment (YinThong) [inline meta-refkit-computervision/recipes-devtools/opengl/python-pyopengl_3.0.1.bb]: Noted. Ok i also try to get python3 with pyopengl.

**Comment by YinThong** [comment]
YES. Please close this PR. Let me consolidated all and send a PR again.

**Pull request closed without merging**